lib/handlers/quote/util/simulation.ts

```diff
@@ -1,22 +1,32 @@
 import { SimulationStatus } from '@uniswap/smart-order-router'
 import Logger from 'bunyan'
 
-export const simulationStatusToString = (simulationStatus: SimulationStatus | undefined, log: Logger) => {
+export enum RoutingApiSimulationStatus {
+  UNATTEMPTED = 'UNATTEMPTED',
+  SUCCESS = 'SUCCESS',
+  FAILED = 'FAILED',
+  INSUFFICIENT_BALANCE = 'INSUFFICIENT_BALANCE',
+  NOT_SUPPORTED = 'NOT_SUPPORTED',
+  NOT_APPROVED = 'NOT_APPROVED',
+  UNKNOWN = '',
+}
+
+export const simulationStatusTranslation = (simulationStatus: SimulationStatus | undefined, log: Logger): RoutingApiSimulationStatus => {
   switch (simulationStatus) {
     case undefined:
-      return 'UNATTEMPTED'
+      return RoutingApiSimulationStatus.UNATTEMPTED
     case SimulationStatus.Succeeded:
-      return 'SUCCESS'
+      return RoutingApiSimulationStatus.SUCCESS
     case SimulationStatus.Failed:
-      return 'FAILED'
+      return RoutingApiSimulationStatus.FAILED
     case SimulationStatus.InsufficientBalance:
-      return 'INSUFFICIENT_BALANCE'
+      return RoutingApiSimulationStatus.INSUFFICIENT_BALANCE
     case SimulationStatus.NotSupported:
-      return 'NOT_SUPPORTED'
+      return RoutingApiSimulationStatus.NOT_SUPPORTED
     case SimulationStatus.NotApproved:
-      return 'NOT_APPROVED'
+      return RoutingApiSimulationStatus.NOT_APPROVED
     default:
       log.error(`Unknown simulation status ${simulationStatus}`)
-      return ''
```

Comment:
there's a behaviorial change from returning empty string '' to UNKNOWN enum type in the response. trading-api team confirmed this is ok. Will check in URA, as well as with web team.

Reply:
web team confirmed this is ok.

Reply:
URA uses `simulationStatus` as pass-through https://github.com/search?q=repo%3AUniswap%2Funified-routing-api%20simulationStatus&type=code

Reply:
all ok